deps/recastnavigation/Detour/Include/DetourMath.h

```diff
@@ -16,5 +19,6 @@ inline float dtMathCeilf(float x) { return ceilf(x); }
 inline float dtMathCosf(float x) { return cosf(x); }
 inline float dtMathSinf(float x) { return sinf(x); }
 inline float dtMathAtan2f(float y, float x) { return atan2f(y, x); }
+inline bool dtMathIsfinite(float x) { return std::isfinite(x); }
```

Comment:
//inline bool dtMathIsfinite(float x) { return std::isfinite(x); }
inline bool dtMathIsfinite(float x) { return isfinite(x); }

Reply:
@pklloveyou what error? I don't understand

Reply:
@FrancescoBorzi Cannot compile

Reply:
Mmpas are being extracted.Let me try.

Reply:
@pklloveyou what compiler do you use and which version?

Reply:
Thank you.

Reply:
@FrancescoBorzi VS2019

Reply:
@Mik1893 can you reproduce this? Because it works fine for me on Ubuntu

Reply:
Selecting Windows SDK version 10.0.18362.0 to target Windows 10.0.14393.
The C compiler identification is MSVC 19.22.27905.0

Reply:
inline bool dtMathIsfinite(float x) { return isfinite(x); }

drop std::